.github/workflows/check-labels.yml

```diff
@@ -1,15 +1,16 @@
 name: check-labels
 on:
-  pull_request:
+  pull_request_target:
     types: [opened, reopened, labeled, unlabeled, synchronize]
   pull_request_review:
     types: [submitted, dismissed]
 jobs:
   check-labels:
     runs-on: ubuntu-latest
     steps:
-    - uses: RIOT-OS/check-labels-action@v1.0.0
+    - uses: RIOT-OS/check-labels-action@master
```

Comment:
using a branch here should work, right?

Reply:
Yepp (but we can also add a new version)

Reply:
Let's see if this works at all or if the `pull_request_event` event is also forced to read-only permissions?

Reply:
I added a v1.1.0 tag now, btw.

Reply:
Ok, changed accordingly.